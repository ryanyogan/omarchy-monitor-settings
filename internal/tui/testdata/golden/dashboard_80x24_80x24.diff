Visual regression detected!

=== EXPECTED ===
  [90m╭────────────────────────────────────────────────────────────────────────╮[0m    
  [90m│[0m                                                                        [90m│[0m    
  [90m│[0m                            [1;34mDisplay Settings[0m                            [90m│[0m    
  [90m│[0m                                                                        [90m│[0m    
  [90m╰────────────────────────────────────────────────────────────────────────╯[0m    
                                                                                
  [34m╭────────────────────────────╮[0m  [36m╭────────────────────────────────────────╮[0m    
  [34m│[0m                            [34m│[0m  [36m│[0m                                        [36m│[0m    
  [34m│[0m                            [34m│[0m  [36m│[0m                                        [36m│[0m    
  [34m│[0m  [1;34mNavigation[0m                [34m│[0m  [36m│[0m  [1;36mDisplay Overview[0m                      [36m│[0m    
  [34m│[0m                            [34m│[0m  [36m│[0m                                        [36m│[0m    
  [34m│[0m  [1;34m▶[0m [1;34mDashboard[0m               [34m│[0m  [36m│[0m  [34m○[0m [1;32mHDMI-A-1[0m[1;33m 👆 CURRENT[0m                 [36m│[0m    
  [34m│[0m                            [34m│[0m  [36m│[0m  [37m  Dell U2414H[0m                         [36m│[0m    
  [34m│[0m    [37mMonitor Selection[0m       [34m│[0m  [36m│[0m  [90m  1920x1080 @ 60Hz[0m                    [36m│[0m    
  [34m│[0m                            [34m│[0m  [36m│[0m  [90m  Scale: 1.0x[0m                         [36m│[0m    
  [34m│[0m    [37mSmart Scaling[0m           [34m│[0m  [36m│[0m  [3;33m  → Scaling changes will apply here[0m   [36m│[0m    
  [34m│[0m                            [34m│[0m  [36m│[0m                                        [36m│[0m    
  [34m│[0m    [37mManual Scaling[0m          [34m│[0m  [36m│[0m  [90m◦[0m [1;34mDP-1[0m                                [36m│[0m    
  [34m│[0m                            [34m│[0m  [36m│[0m  [37m  Samsung C27F390[0m                     [36m│[0m    
  [34m│[0m    [37mSettings[0m                [34m│[0m  [36m│[0m  [90m  1920x1080 @ 75Hz[0m                    [36m│[0m    
  [34m│[0m                            [34m│[0m  [36m│[0m  [90m  Scale: 1.2x[0m                         [36m│[0m    
  [34m│[0m    [37mHelp[0m                    [34m│[0m  [36m│[0m                                        [36m│[0m    
  [34m│[0m                            [34m│[0m  [36m│[0m                                        [36m│[0m    
  [34m│[0m    [37mExit[0m                    [34m│[0m  [36m│[0m                                        [36m│[0m    
  [34m│[0m                            [34m│[0m  [36m╰────────────────────────────────────────╯[0m    
  [34m│[0m                            [34m│[0m                                                
  [34m│[0m                            [34m│[0m                                                
  [34m╰────────────────────────────╯[0m                                                
                                                                                
  [90m╭────────────────────────────────────────────────────────────────────────╮[0m    
  [90m│[0m                                                                        [90m│[0m    
  [90m│[0m       [1;32m↑↓[0m  [37mnavigate[0m    [1;34m⏎[0m  [37mselect[0m    [1;33mh[0m  [37mhelp[0m    [1;35mesc[0m  [37mback[0m    [1;31mq[0m  [37mquit[0m     [90m│[0m    
  [90m│[0m                                                                        [90m│[0m    
  [90m╰────────────────────────────────────────────────────────────────────────╯[0m    

=== ACTUAL ===
  ╭────────────────────────────────────────────────────────────────────────╮    
  │                                                                        │    
  │                            Display Settings                            │    
  │                                                                        │    
  ╰────────────────────────────────────────────────────────────────────────╯    
                                                                                
  ╭────────────────────────────╮  ╭────────────────────────────────────────╮    
  │                            │  │                                        │    
  │                            │  │                                        │    
  │  Navigation                │  │  Display Overview                      │    
  │                            │  │                                        │    
  │  ▶ Dashboard               │  │  ○ HDMI-A-1 👆 CURRENT                 │    
  │                            │  │    Dell U2414H                         │    
  │    Monitor Selection       │  │    1920x1080 @ 60Hz                    │    
  │                            │  │    Scale: 1.0x                         │    
  │    Smart Scaling           │  │    → Scaling changes will apply here   │    
  │                            │  │                                        │    
  │    Manual Scaling          │  │  ◦ DP-1                                │    
  │                            │  │    Samsung C27F390                     │    
  │    Settings                │  │    1920x1080 @ 75Hz                    │    
  │                            │  │    Scale: 1.2x                         │    
  │    Help                    │  │                                        │    
  │                            │  │                                        │    
  │    Exit                    │  │                                        │    
  │                            │  ╰────────────────────────────────────────╯    
  │                            │                                                
  │                            │                                                
  ╰────────────────────────────╯                                                
                                                                                
  ╭────────────────────────────────────────────────────────────────────────╮    
  │                                                                        │    
  │       ↑↓  navigate    ⏎  select    h  help    esc  back    q  quit     │    
  │                                                                        │    
  ╰────────────────────────────────────────────────────────────────────────╯    

=== DIFF INFO ===
Expected length: 4992
Actual length: 3821
Expected hash: 8b88f2b9da086100814b5c6794ea55a28c2aa5fc9b09426120432876e763851a
Actual hash: b230c4779bd5431702336f51ffcd03329ebdefc247bd9539be4cee81b5a54331
