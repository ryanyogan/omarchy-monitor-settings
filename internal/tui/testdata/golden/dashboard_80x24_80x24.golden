# Visual Golden File
# Name: dashboard_80x24
# Dimensions: 80x24
# Hash: b230c4779bd5431702336f51ffcd03329ebdefc247bd9539be4cee81b5a54331

  ╭────────────────────────────────────────────────────────────────────────╮    
  │                                                                        │    
  │                            Display Settings                            │    
  │                                                                        │    
  ╰────────────────────────────────────────────────────────────────────────╯    
                                                                                
  ╭────────────────────────────╮  ╭────────────────────────────────────────╮    
  │                            │  │                                        │    
  │                            │  │                                        │    
  │  Navigation                │  │  Display Overview                      │    
  │                            │  │                                        │    
  │  ▶ Dashboard               │  │  ○ HDMI-A-1 👆 CURRENT                 │    
  │                            │  │    Dell U2414H                         │    
  │    Monitor Selection       │  │    1920x1080 @ 60Hz                    │    
  │                            │  │    Scale: 1.0x                         │    
  │    Smart Scaling           │  │    → Scaling changes will apply here   │    
  │                            │  │                                        │    
  │    Manual Scaling          │  │  ◦ DP-1                                │    
  │                            │  │    Samsung C27F390                     │    
  │    Settings                │  │    1920x1080 @ 75Hz                    │    
  │                            │  │    Scale: 1.2x                         │    
  │    Help                    │  │                                        │    
  │                            │  │                                        │    
  │    Exit                    │  │                                        │    
  │                            │  ╰────────────────────────────────────────╯    
  │                            │                                                
  │                            │                                                
  ╰────────────────────────────╯                                                
                                                                                
  ╭────────────────────────────────────────────────────────────────────────╮    
  │                                                                        │    
  │       ↑↓  navigate    ⏎  select    h  help    esc  back    q  quit     │    
  │                                                                        │    
  ╰────────────────────────────────────────────────────────────────────────╯    